molecule_ec2/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -73,53 +73,49 @@
         filters:
           name: "{{ item.image_name }}"
       loop: "{{ molecule_yml.platforms }}"
-      when: item.image is not defined
+      when: item.image_id is not defined
```

Comment:
Shifting from `item.image` to `item.image_id` doesn't seem worthwhile.

It will require updates to the documentation (not included) and break compatibility with older molecule.yml files

I can see that it brings consistency with the AWS calls but that feels like an insufficient reason to break with the standard interface.